From 199d6fd8ab2e9095b45d0258841247a21c3786b9 Mon Sep 17 00:00:00 2001
From: HiassofT <github@hias.horus.com>
Date: Wed, 6 Apr 2016 21:45:01 +0200
Subject: [PATCH] Revert "bcm2835-dma: Fix dreq not set for slave transfers"

This reverts commit 8ad957e866a1fe1450f663f2b00a57d7de44904c.



DMA channels are set in devicetree, thus dreq will be set,

and this commit is no longer needed.



Signed-off-by: Matthias Reichl <hias@horus.com>
---
 drivers/dma/bcm2835-dma.c | 2 --
 1 file changed, 2 deletions(-)

--- a/drivers/dma/bcm2835-dma.c
+++ b/drivers/dma/bcm2835-dma.c
@@ -679,8 +679,6 @@ static int bcm2835_dma_slave_config(stru
 	}
 
 	c->cfg = *cfg;
-	if (!c->dreq)
-		c->dreq = cfg->slave_id;
 
 	return 0;
 }
